pilot/model/validation.go

```diff
@@ -852,6 +883,70 @@ func ValidateIngressRule(msg proto.Message) error {
 	return errs
 }
 
+// ValidateGateway checks ingress rules
```

Comment:
nit: checks gateway rules

Reply:
What do we want to validate here? Rules aren't directly embedded in the Gateway spec. We can check, when a gateway is created, that there are some route rules that reference it, but I think the better way would be to validate at RouteRule ingestion that the RouteRule references a valid Gateway. Either way I think we'll want to allow both Gateways with no RouteRules bound to them, and RouteRules that bind to non-extant Gateways.

Reply:
Sorry for the mis understanding..May be say "validategateway checks gateway configuration" ? rule is probably a wrong word to use..

Reply:
Oh whoops, sorry, totally misread your comment! Yes, comment string is a copy/paste error on my part, thanks.